src/EventModels.ts

```diff
@@ -0,0 +1,150 @@
+export enum ThumbnailSupport {
+    START_END = 0,
+    START_WITH_DELAYED_END,
+    SINGLE_DELAYED,
+    SINGLE,
+    NONE,
+}
+
+export type Metadata =
+    | {
+          label: string
+          id: string
+          hasDuration: false
+          valueExpression: string
+          thumbnailSupport: ThumbnailSupport
+      }
+    | {
+          label: string
+          id: string
+          hasDuration: boolean
+          valueExpression?: undefined
+          thumbnailSupport: ThumbnailSupport
+      }
+
+export type UnifiEventModel = {
+    shapeProfile: any
```

Comment:
We do not like `any` :)

Reply:
`shapeProfile` can be any shape...
And is deigned to be dynamic (so it can match any object type)

else - we need to define a type that covers all possible events from unify?

Reply:
@Shaquu & @crxporter 
I have moved from  `any` to `Record<string, any>` to identify these objects as JSON objects
this is for the bootstrap and `shapeProfile`.